packages/diagram/lib/diagram.dart

```diff
@@ -0,0 +1,442 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+library diagram;
```

Comment:
we generally don't include the library line unless we're documenting the library itself

Reply:
So.. I'm not sure what you mean here.  This is the library itself, as far as I'm aware.

Reply:
Right but you don't have any dartdocs for the `library`. So you can just omit the `library` line.

Reply:
Ahh, so it's useless for anything other than attaching docs to?  I didn't know that.  Removed.